chains/ethereum/src/api.ts

```diff
@@ -1,76 +1,15 @@
 import type { Connector } from '@wagmi/core'
-import {
-  connect,
-  disconnect,
-  fetchBalance,
-  fetchEnsAddress,
-  fetchEnsAvatar,
-  fetchEnsName,
-  fetchEnsResolver,
-  fetchSigner,
-  fetchToken,
-  fetchTransaction,
-  getContract,
-  getNetwork,
-  InjectedConnector,
-  prepareSendTransaction,
-  prepareWriteContract,
-  readContract,
-  sendTransaction,
-  signMessage,
-  signTypedData,
-  switchNetwork,
-  waitForTransaction,
-  watchReadContract,
-  writeContract
-} from '@wagmi/core'
-import { CoinbaseWalletConnector } from '@wagmi/core/connectors/coinbaseWallet'
-import { MetaMaskConnector } from '@wagmi/core/connectors/metaMask'
-import { WalletConnectConnector } from '@wagmi/core/connectors/walletConnect'
-import { jsonRpcProvider } from '@wagmi/core/providers/jsonRpc'
+import * as WagmiCore from '@wagmi/core'
```

Comment:
Shouldn't be an issue with modern tools like rollup. It's called `Static Import Full/Fully`. It would be an issue if there are default exports there though.